xlgui/widgets/smart_playlist_editor.py

```diff
@@ -254,7 +256,7 @@ def __init__(self):
     N_('Playlist'): '__playlist',  # not a real tag
 }
 
-_REV_NMAP = {}
+_REV_NMAP: Mapping[str, str] = {}
```

Comment:
I think `Dict` would be more suitable here.

Reply:
`Mapping` is more generic than `Dict`, which could be nice sometimes, but as the code creates a `Dict`, that's probably better.